package.json

```diff
@@ -1,54 +1,49 @@
 {
   "name": "ember-singularity",
-  "version": "1.0.2",
-  "description": "Unified event handling for Ember.",
+  "version": "0.0.0",
+  "description": "The default blueprint for ember-cli addons.",
```

Comment:
Need to bring back some of the original package.json contents (author, version, description, keywords, repository).